app/assets/javascripts/browsered/web-payments/format-card-types.js

```diff
@@ -8,6 +8,8 @@ module.exports = (allowedCardTypes, provider) => {
   const availableNetworks = allowedCardTypes.map(type => {
     if (type.debit || type.credit) {
       return type.brand
+    } else {
+      return []
```

Comment:
I think we may need we change this to return `false` or `null`.
Otherwise it will return true when a empty [] is returned.

Reply:
I'm going to break this into another PR as this one is quite large but I think Array.prototype.map() expects a value to be returned at the end of an arrow function, returning `false` / `null` here would break the conditionals further down the function.

Reply:
Resolved as no longer part of this PR